wildfly/feature-pack/src/main/resources/modules/org/infinispan/slot/module.xml

```diff
@@ -83,20 +84,21 @@
 
       <module name="org.apache.xerces" services="import"/>
 
-      <module name="org.eclipse.microprofile.config.api" export="true"/>
-      <module name="org.eclipse.microprofile.metrics.api" export="true"/>
-      <module name="io.smallrye.config" services="export" export="true"/>
-      <module name="io.smallrye.metrics" services="import" export="true"/>
+      <module name="org.eclipse.microprofile.config.api" export="true" optional="true"/>
+      <module name="org.eclipse.microprofile.metrics.api" export="true" optional="true"/>
+      <module name="io.smallrye.config" services="export" export="true" optional="true"/>
+      <module name="io.smallrye.metrics" services="import" export="true" optional="true"/>
 
       <module name="org.jboss.jandex"/>
       <module name="org.jboss.logging"/>
-      <module name="org.jboss.weld.core" />
-      <module name="org.jboss.weld.spi" />
+      <module name="org.jboss.weld.core"/>
+      <module name="org.jboss.weld.spi"/>
       <module name="org.wildfly.security.elytron" services="import"/>
       <module name="org.hibernate"/>
+      <module name="org.hibernate.commons-annotations"/>
```

Comment:
Will not help for older EAP versions and might have issues as EAP might use legacy hibernate* libraries.
Not sure whether ISPN will work properly.
I would expect that we ship the artifacts we need